pkg/resource/resource_test.go

```diff
@@ -0,0 +1,389 @@
+package resource_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/types"
+	"github.com/openshift-psap/special-resource-operator/pkg/resource"
+	buildv1 "github.com/openshift/api/build/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func TestResource(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Resource Suite")
+}
+
+var _ = Describe("Resource", func() {
+	Describe("IsNamespaced", func() {
```

Comment:
2 levels of Describes are not a very good practice. Lets' replace it with Context

Reply:
Removed ` var _ = Describe("Resource", func() {` as it's unnecessary here.